### DISCUSSION TITLE
Paypal transactions API retrieval and hledger-flow processing

### DISCUSSION THREAD

**Discussion by sbibauw** [apauley/hledger-flow#101] [Show and tell]
For anyone looking to import Paypal transactions, I've put on https://gist.github.com/sbibauw/6cc0c36cc22a857a3237b977515318bb a full workflow including:
- API retrieval of current & previous month transactions (returns a JSON)
- `preprocess` shell script for transforming the JSON into a CSV
- `paypal.rules` file

**Comment by apauley** [comment]
Thanks @sbibauw! 😎
